src/logic/safe/transactions/gas.ts

```diff
@@ -147,6 +130,11 @@ const getGasEstimationTxResponse = async (txConfig: {
 
     if (!sameString(result, EMPTY_DATA)) {
       return new BigNumber(result.substring(138), 16).toNumber()
+    } else {
+      // This is a transfer of native coin (eth, xDai, etc), in that case the empty result is valid
+      if (sameString(txConfig.data, EMPTY_DATA)) {
+        return new BigNumber(21000).toNumber()
```

Comment:
Could this be just 21000? 🙂

Reply:
As mentioned above, we should not have this if branch, as it doesn't make sense in the context of a Safe transaction. 

Even in the normal EOA ethereum context you should never assume that a transfer is only 21k gas. E.g. if your receiving address is a contract the gas costs will always be higher.

Reply:
Thanks for the feedback guys, we have worked together with @fernandomg and @nicosampler (thanks guys) and ended up with just:

```
    const estimateGas = await estimateExecTransactionGas(safeAddress, data, txTo, txAmount, operation)

    return estimateGas <= 0
```

As just @rmeissner suggested 💪

That line was removed, not needed anymore